odoo/addons/openupgrade_records/lib/apriori.py

```diff
@@ -26,15 +26,19 @@
 
 renamed_models = {
     # Odoo
+    'hr.holidays': 'hr.leave',
+    'hr.holidays.status': 'hr.leave.type',
+    'mrp.repair': 'repair.order',
+    'mrp.repair.fee': 'repair.fee',
+    'mrp.repair.line': 'repair.line',
     'procurement.rule': 'stock.rule',
     'product.attribute.line': 'product.template.attribute.line',
     'product.attribute.price': 'product.template.attribute.value',
     'product.uom': 'uom.uom',
     'product.uom.category': 'uom.category',
-    'hr.holidays': 'hr.leave',
-    'hr.holidays.status': 'hr.leave.type',
-    'sale.quote.template': 'sale.order.template',
-    'sale.quote.option': 'sale.order.template.option',
     'sale.quote.line': 'sale.order.template.line',
+    'sale.quote.option': 'sale.order.template.option',
+    'sale.quote.template': 'sale.order.template',
     'stock.incoterms': 'account.incoterms',
+    'stock.location.path': 'stock.rule',
```

Comment:
I don't know if this will give problems. Having two models renamed into a new one.

Reply:
Yes. You should rename one of them, and merge the other one in this. Any way, is this model renaming being called in base module right now?

Reply:
No, it's just only used in the openupgrade analysis.

Reply:
The error would raise there, but any way, I think it's not a good idea to put both for the analysis.

Reply:
Ok, I commented it instead.